test/BlobTest.cc

```diff
@@ -69,6 +69,7 @@ TEST_CASE_METHOD(BlobTest, "Create blob stream and close", "[Blob]") {
 TEST_CASE_METHOD(BlobTest, "Create blob with stream", "[Blob]") {
     static constexpr slice kBlobContent = "This is the content of the blob 1.";
     CBLError error;
+    CBLCollection *coll = CBLDatabase_DefaultCollection(db, &error);
```

Comment:
It seems that this is going to happen a lot, so you may consider making defaultCollection a member variable along side `db`

Reply:
Nevermind, I was looking at the first commit, seems this is changed.